web-workers/fibonacci-worker/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,15 @@
+# Community Participation Guidelines
```

Comment:
We will not need this anymore as everything in this repo is covered by https://github.com/mdn/dom-examples/blob/master/CODE_OF_CONDUCT.md